firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -381,6 +436,10 @@ public synchronized void beginRealtimeHttpStream() {
       if (responseCode == null
           || responseCode == HttpURLConnection.HTTP_OK
           || isStatusCodeRetryable(responseCode)) {
+        if (responseCode == null || isStatusCodeRetryable(responseCode)) {
```

Comment:
Unnesting this conditional would be easier to read since it partially overlaps with the parent condition

Reply:
Simplified it to remove the nesting